mythril/laser/ethereum/transaction/symbolic.py

```diff
@@ -23,7 +23,7 @@ def execute_message_call(laser_evm, callee_address):
             callee_account=open_world_state[callee_address],
             caller=BitVec("caller{}".format(next_transaction_id), 256),
             identifier=next_transaction_id,
-            call_data=[],
+            call_data=Calldata(next_transaction_id),
```

Comment:
Shouldn't there be some set_global_state() happening somewhere?

Reply:
It's happening. In the `initial_global_state` function of MessageCallTransaction.

Reply:
It's happening. In the initial_global_state function of MessageCallTransaction.